src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5173,6 +5175,89 @@ namespace HeatBalanceManager {
         GetSpaceData(state, ErrorsFound);
     }
 
+    void GetIncidentSolarMultiplier(EnergyPlusData &state, bool &ErrorsFound)
+    {
+        auto &cCurrentModuleObject = state.dataIPShortCut->cCurrentModuleObject;
+        cCurrentModuleObject = "SurfaceProperty:IncidentSolarMultiplier";
+
+        constexpr const char *RoutineName("GetIncidentSolarMultiplier: ");
```

Comment:
Should probably be made `string_view`. I assume we want to move away from `const char *`.